src/gui/DatabaseWidgetStateSync.cpp

```diff
@@ -25,14 +25,16 @@ DatabaseWidgetStateSync::DatabaseWidgetStateSync(QObject* parent)
     , m_activeDbWidget(nullptr)
     , m_blockUpdates(false)
 {
-    m_splitterSizes = variantToIntList(config()->get("GUI/SplitterState"));
+    m_mainSplitterSizes = variantToIntList(config()->get("GUI/SplitterState"));
```

Comment:
@TheZ3ro maybe it makes sense to rename the config variable to "GUI/MainSplitterState"?

Reply:
Changing this will "break" all old configuration. Keep it the same, not a big deal

Reply:
We can use the migration code, like this: 

```cpp
 m_mainSplitterSizes = variantToIntList(config()->get("GUI/MainSplitterState"));
 if (m_mainSplitterSizes.isEmpty()) {
   // try to read from old config variable
    m_mainSplitterSizes = variantToIntList(config()->get("GUI/splitterState"));
 }
```

Reply:
I don't really like adding more code for such a thing. 

IMHO if we want break it's not a big deal, users can re-configure their window split.
But at the same time using `GUI/MainSplitterState` instead of `GUI/splitterState` it's not a big deal as well.

Reply:
ok. leave everything as it is at the moment.